packages/gatsby-plugin-styletron/src/gatsby-browser.js

```diff
@@ -1,13 +1,6 @@
-const React = require(`react`)
-const Styletron = require(`styletron-engine-atomic`).Client
-const StyletronProvider = require(`styletron-react`).Provider
+import { createElement as h } from "react"
```

Comment:
It can be written as JSX if that's what's desired, I generally write code this way because I try to write code that can be run in the browser (without babel, [example](http://ptb2.me/simpleReactCounter/)). In this case, it's not necessary (obviously), and I don't have a strong opinion one way or another.

Reply:
Oh, I don't mean to write it as JSX - just aliasing `createElement` as `h` and not just using `createElement` as later when people will read that code will have to figure out what `h` means and `createElement` is pretty known react function :)

Reply:
I think it's something I picked up from preact and/or hyperscript.